tools/jil/util/browser-list.js

```diff
@@ -148,7 +148,7 @@ function byVersion(left, right) {
 function cleanVersion(version) {
   // assign to high number, so that it is high in the list when sorted (i.e. beta is highest)
   if (!version || latestVersStringRe.test(version)) {
-    let prevVersionOffset = version?.split('-')[1];
+    let prevVersionOffset = !!version && version.split('-')[1];
```

Comment:
Q: Out of curiosity, what's wrong with the optional chaining? It's the same thing and works in Node 14.

Reply:
Jil was throwing errors with it for some reason at some point in the months-long POC 😓